### ISSUE TITLE
Migrate to use upstream GraphQL li 

### ISSUE THREAD

**Issue by wtrocki** [aerogear/graphql-link#41]
Currently we are using Chirino fork of the GraphQL. 
GraphQL implementation moved forward we should investigate how much work will be needed to migrate

**Comment by ankithans** [comment]
Can I start working on it? 
There are some extra files and folders in Chirino's fork. If we switch to upstream then there will be issues ig

**Comment by ankithans** [comment]
The version which we are using has some additional go modules but the upstream version doesn't have it. And those modules are used widely in the project. So what should I do for them now?

Chirino version was forked at this point and then he added modules over it https://github.com/graph-gophers/graphql-go/tree/25d6d94fa7a7f2b0ce10fd509e54e85f7a2f866b

<table>
<tr>
<td>our</td>
<td>upstream</td>
</tr>
<tr>
<td><img src="https://user-images.githubusercontent.com/50692897/109642375-e7ca0300-7b78-11eb-9459-eb1b0e5772c8.png" /></td>
<td><img src="https://user-images.githubusercontent.com/50692897/109642407-eef11100-7b78-11eb-8723-33548fb6498e.png" /></td>
</tr>
</table>

**Comment by wtrocki** [comment]
Ok. I'm aware of this, but what you expect me to do with this info? What is your question?
I would start with fixing CORS or JSON parse issues to get more familiar with the project.

**Comment by ankithans** [comment]
So my question is when I replace the imports everywhere from ```github.com/chirino/graphql``` to ```github.com/graph-gophers/graphql-go```, there will be errors coming up for the modules which are not there.

So how should I include those modules in our project? Like should I add them as modules in ```graphql-link``` ?

**Comment by wtrocki** [comment]
This is low level implementation question. I guess you need those from somewhere.

**Comment by ankithans** [comment]
Yeah, they are available in ```github.com/chirino/graphql```. I can add and update the necessary ones in ```graphql-link```. If that sound fine?

**Comment by wtrocki** [comment]
Not sure what you mean TBH. Let's create PR with explanation what was done.

**Comment by ankithans** [comment]
Okay sure @wtrocki, will create PR soon. After working on the issue, I will also have more clarity.

**Comment by wtrocki** [comment]
Trust me - it will not be soon. There is more than week of work on migration IMHO. 
Please try some easiest issues first

**Comment by ankithans** [comment]
yeah realizing 🥲
> Please try some easiest issues first

okay, got it!

**Comment by namit-chandwani** [comment]
@wtrocki Can you please assign this issue to me? Will start working on it right away
